### ISSUE TITLE
Move scrollbar to the left

### ISSUE THREAD

**Issue by hkkmori** [QupZilla/qupzilla#2160]
How can we move the scrollbar to the left ?
There must be a qt workaround ..if possible without recompiling.

Thanks.

**Comment by hkkmori** [comment]
Could you add this parameter in the next release , please ?

**Comment by nowrep** [comment]
No, RTL language (Right-To-Left) means that all of the GUI is reversed, not only scrollbar position.

**Comment by hkkmori** [comment]
If i understand right , it would be too much work to use RTL language now ?
Instead could you add an extra scrollbar to the left ? And also a option to hide the right-side scrollbar.

Sorry for not being a qt specialist .

**Comment by hkkmori** [comment]
up

**Comment by nowrep** [comment]
This is what I mean with RTL language - https://en.wikipedia.org/wiki/Right-to-left

If you don't use such language then you don't want to use reversed GUI.
There is a switch to force RTL - `qupzilla -reverse`

**Comment by hkkmori** [comment]
I don't want to use reversed GUI just to have the scrollbar to the left .
Why is it so complicated with qt ?

**Comment by hkkmori** [comment]
up

**Comment by nowrep** [comment]
It's not possible with Qt.
